packages/nouns-contracts/contracts/libs/Noracle.sol

```diff
@@ -93,20 +93,17 @@ library Noracle {
         uint16 initializedObservationsFound = 0;
         uint16 checkedIndexesCount = 0;
         while (initializedObservationsFound < fromAuctionsAgo && checkedIndexesCount < cardinality) {
-            checkedIndexesCount++;
             uint16 checkIndex = (self.index + (cardinality - checkedIndexesCount)) % cardinality;
             Observation storage obs = self.observations[checkIndex];
             if (obs.initialized) {
                 observations[initializedObservationsFound] = obs;
                 initializedObservationsFound++;
```

Comment:
quite minor, but I believe `++i` is cheaper in gas than `i++`

Reply:
changed as part of this change https://github.com/nounsDAO/nouns-monorepo/pull/598/commits/b8557264a97216dfe1bb35a9c8705a0439fe35eb